deps.ts

```diff
@@ -1 +1,2 @@
 export type { Context, Middleware } from "https://deno.land/x/oak/mod.ts";
+export { connect } from "https://deno.land/x/redis@v0.25.0/mod.ts"
```

Comment:
RedisStore should contain all its dependencies in its own file since not everyone would want to cache its dependencies if they're not using it.